src/mono/wasm/debugger/BrowserDebugProxy/DevToolsHelper.cs

```diff
@@ -92,6 +93,9 @@ public static bool TryParse(string id, out DotnetObjectId objectId)
                         parts = id.Split(":");
                         if (parts.Length > 3)
                             objectId.SubValue = int.Parse(parts[3]);
+                        ElementType et;
+                        Enum.TryParse(parts[4], out et);
```

Comment:
When using `Try*` methods, we should check for the return value to see if it actually succeeded. In this case, if it returns `false`, then return `false` from the method.